README.markdown

```diff
@@ -641,6 +642,41 @@ func reticulateSplines(spline: [Double], adjustmentFactor: Double,
 
 ```
 
+ For functions with very long signatures, you can place each parameter on its own line with normal indentation to improve readability:
+ 
+
+```swift
+func reticulateSplines(
+    spline: [Double],
+    adjustmentFactor: Double,
+    translateConstant: Int,
+    rollTransform: Double,
+    pitchTransform: Double,
+    yawTransform: Double,
+    comment: String
+) -> Bool {
```

Comment:
I really like this style. I go so far as to put everything on it's own line as soon as it becomes longer than one line. That usually translates to three parameters.